dvc/utils/studio.py

```diff
@@ -125,3 +126,19 @@ def get_subrepo_relpath(repo: "Repo") -> str:
     relpath = as_posix(repo.fs.relpath(repo.root_dir, scm_root_dir))
 
     return "" if relpath == "." else relpath
+
+
+def get_repo_url(repo: "Repo") -> str:
```

Comment:
[Q] There is another call to `get_remote_repo` in `notify_refs_to_studio`, should that get moved onto this util as well?

Reply:
LGTM